dtls.go

```diff
@@ -0,0 +1,5 @@
+package mint
+
+const (
+	initialMtu = 1200
+)
```

Comment:
Nit: It would be good to have a comment in this file noting that it's a stub, and what else is supposed to go here